src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts

```diff
@@ -107,6 +122,13 @@ export class ResultsFiltersPreferences extends Component {
   private advancedFiltersTabSelect: PreferencePanelMultiSelectInput;
   private advancedFilterFormValidate: HTMLFormElement;
 
+  /**
+   * Creates a new ResultsFiltersPreferences component.
+   * @param element The HTMLElement on which to instantiate the component.
+   * @param options The options for the ResultsFiltersPreferences component.
+   * @param bindings The bindings that the component requires to function normally. If not set, these will be
+   * automatically resolved (with a slower execution time).
+   */
   constructor(public element: HTMLElement, public options: IResultsFiltersPreferencesOptions, public bindings: IComponentBindings) {
     super(element, ResultsFiltersPreferences.ID, bindings);
 
```

Comment:
Should I expose the `createDom()`, `save()` and `exitWithoutSave()` public methods?